src/main/java/net/galgus/flink/streaming/connectors/http/HTTPSink.java

```diff
@@ -38,6 +38,11 @@ public void invoke(IN value, Context context) throws Exception {
             writer.close();
 
             int status = conn.getResponseCode();
+            log.info("Status={}", status);
+            if(status == 200) {
```

Comment:
there are other successful status codes such as 201, a better check is to see if there is a method to do this already in the conn object

Reply:
The conn object does not provide any method to check successful status codes. so just edited the if statement to take into account all 200 codes